docs/UserGuide.md

```diff
@@ -238,27 +239,56 @@ to `91234567` and `johndoe@example.com` respectively.
 [To be filled up]
 ### Finding Patients
 
-#### Listing all persons : `list`
+{: .alert .alert-info}
+> :information_source: **Note:**
+> 
+> * The following commands in this section will alter the view of the Patient List Panel to display only patients that match the given criteria.
+> * They will not alter the view of the Appointment List Panel.
 
-Shows a list of all persons in the address book.
+#### Listing all patients : `list`
+
+Shows a list of all patients in MediBase3.
 
 Format: `list`
 
+{: .alert .alert-success}
+> :bulb: **Tip:**
+>
+> If you have used any other commands under the [Finding Patients](#finding-patients) section to alter the view of the Patient List Panel, 
+> you can use this command to reset to the default view to view all patients!
+
```

Comment:
good catch on possible situations user might find themselves in. Also helps them if they ctrl-f and search